### DISCUSSION TITLE
How should I add a new backend in einops ?

### DISCUSSION THREAD

**Discussion by rentainhe** [arogozhnikov/einops#149] [Ideas]
Hello! How should I add a new backend in this repo~, actually I want to add the [OneFlow](https://github.com/Oneflow-Inc/oneflow) backend in einops

**Comment by arogozhnikov** [comment]
Hey @rentainhe,
you can introduce einops for a new backend, all you need is to define backend for oneflow, see examples of backend in backend.py. 

Just define one like that in a file (e.g. `einops_for_oneflow.py`) and import that file before using einops operations.
Einops will (magically) recognize presence of a new backend and operations will work for the new backend, 
see #120 for some discussion.
